src/components/coverage-date-list/coverage-date-list.js

```diff
@@ -3,9 +3,35 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import { FormattedDate, FormattedMessage } from 'react-intl';
 
+/* eslint-disable consistent-return */
```

Comment:
Does placing `return {}` at the end of `containsNonEmptyObjectsWithStringValues()` work? That might be preferable to the `eslint-disable` declaration.

Reply:
Have deleted `eslint-disable` and added `return null` at the end.

Reply:
Jeffrey, are you OK with this?

Reply:
👍 yup, that's better.